teamB/kimSungHoon/controllers/userController.js

```diff
@@ -0,0 +1,53 @@
+const userService = require('../services/userService');
+
+const signUp = async (req, res) => {
+    try {
+        const { name, email, password, profile_image } = req.body;
+
+        if( !name || !email || !password || !profile_image ){   
+            return res.status(400).json({message: '키 에러 발생'});
+        }
+
+        await userService.signUp(name, email, password, profile_image);
+        return res.status(201).json({message: '유저 회원 가입 성공'});
+    }
+    catch(err){
+        console.log(err);
+        return res.status(err.statusCode || 500).json({message: err.message});
+    }
+};
+
+const userPosting = async (req, res) => {
+    try {
+        const id = req.params.id;
+        const result = await userService.userPosting(id);
+        return res.status(201).json({ data: result });
+    }
+    catch(err){
+        console.log(err);
+        return res.status(err.statusCode || 500).json({message: err.message});
+    }
+}
+
+const like = async (req, res) => {
+    try {
+        const {user_id, post_id} = req.body;
+
+        if( !user_id || !post_id ){   
+            return res.status(400).json({message: '키 에러 발생'});
+        }
+
+        const result = await userService.like(user_id, post_id);
+        return res.status(201).json({ "message" : "likeCreated" });
+    }
+    catch(err){
+        console.log(err);
+        return res.status(err.statusCode || 500).json({message: err.message});
+    }
+}
+
+module.exports = {
+	signUp,
+    userPosting,
+    like
+}
```

Comment:
- 해당 부분도 indentation을 지켜주세요 !
```suggestion
module.exports = {
    signUp,
    userPosting,
    like
}
```

Reply:
원본 파일에 indentation을 지켜서 올렸는데 github 상에서는 들여쓰기가 되어있는 것으로 나옵니다... 그리고 git 업을 해도 수정이 안되는 상황입니다. 위에 말씀해주신 부분도 동일합니다.